requirements.txt

```diff
@@ -37,3 +37,4 @@ tqdm==4.64.0
 nicegui==1.2.16
 transformers==4.29.2
 python-dotenv==1.0.0
+ctransformers
```

Comment:
It will be great if we put this at its alphabetical place when it comes to ordering in requirements :)